packages/app-store/routing-forms/pages/form-edit/[...appPages].tsx

```diff
@@ -50,431 +38,40 @@ const appendArray = <T,>({
-  });
-
-  const [animationRef] = useAutoAnimate<HTMLDivElement>();
-
-  const addField = () => {
-    appendHookFormField({
-      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-      //@ts-ignore
-      id: uuidv4(),
-      // This is same type from react-awesome-query-builder
-      type: "text",
-      label: "",
-    });
-  };
-
-  // hookForm.reset(form);
-  if (!form.fields) {
-    form.fields = [];
-  }
-  return hookFormFields.length ? (
-    <div className="flex flex-col-reverse lg:flex-row">
-      <div className="w-full ltr:mr-2 rtl:ml-2">
-        <div ref={animationRef} className="flex w-full flex-col rounded-md">
-          {hookFormFields.map((field, key) => {
-            return (
-              <Field
-                appUrl={appUrl}
-                fieldIndex={key}
-                hookForm={hookForm}
-                hookFieldNamespace={`${fieldsNamespace}.${key}`}
-                deleteField={{
-                  check: () => hookFormFields.length > 1,
-                  fn: () => {
-                    removeHookFormField(key);
-                  },
-                }}
-                moveUp={{
-                  check: () => key !== 0,
-                  fn: () => {
-                    swapHookFormField(key, key - 1);
-                  },
-                }}
-                moveDown={{
-                  check: () => key !== hookFormFields.length - 1,
-                  fn: () => {
-                    if (key === hookFormFields.length - 1) {
-                      return;
-                    }
-                    swapHookFormField(key, key + 1);
-                  },
-                }}
-                key={key}
-              />
-            );
-          })}
-        </div>
-        {hookFormFields.length ? (
-          <div className={classNames("flex")}>
-            <Button
-              data-testid="add-field"
-              type="button"
-              StartIcon="plus"
-              color="secondary"
-              onClick={addField}>
-              Add field
-            </Button>
-          </div>
-        ) : null}
-      </div>
-    </div>
-  ) : (
-    <div className="bg-default w-full">
-      <EmptyScreen
-        Icon="file-text"
-        headline="Create your first field"
-        description="Fields are the form fields that the booker would see."
-        buttonRaw={
-          <Button data-testid="add-field" onClick={addField}>
-            Create Field
-          </Button>
-        }
-      />
-    </div>
-  );
-};
-
 export default function FormEditPage({
   appUrl,
   ...props
 }: inferSSRProps<typeof getServerSideProps> & { appUrl: string }) {
+  const { t } = useLocale();
+  const formMethods = useFormContext<FormValues>();
+
   return (
     <SingleForm
       {...props}
       appUrl={appUrl}
-      Page={({ hookForm, form }) => <FormEdit appUrl={appUrl} hookForm={hookForm} form={form} />}
+      Page={({ hookForm, form }) => (
+        <>
+          <FormBuilder
```

Comment:
This is great to see that it reduced the file size a lot !!

But we need to ensure that all functionalities that were supported in Form-edit work. If there is some not-critical functionality that doesn't work, let's make a list and then we can decide if it can be fixed in a followup or required to be fixed in this PR itself